.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -184,3 +188,54 @@ private object InlineHelp : WPComPluginBuild(
 	docsLink = "TODO",
 	withPRNotify = "false",
 )
+
+private object GutenbergUploadSourceMapsToSentry: BuildType() {
+	init {
+		name = "Upload Gutenberg Source Maps to Sentry";
+		id("WPComPlugins_GutenbergUploadSourceMapsToSentry");
+
+		// vcs {
+		//	GitVcsRoot {
+		//		name = "gutenberg"
+		//		url = "git@github.com:fullofcaffeine/gutenberg.git"
+		//		pushUrl = "git@github.com:fullofcaffeine/gutenberg.git"
+		//		branch = "refs/heads/trunk"
+		//		branchSpec = "+:refs/heads/*"
+		//		useTagsAsBranches = true
+		//		authMethod = uploadedKey {
+		//			uploadedKey = "matticbot"
+		//		}
+		//	}
+		// }
+
+		params {
+			text(
+				name = "GUTENBERG_VERSION",
+				value = "",
+				label = "Gutenberg version",
+				description = "The Gutenberg version to upload source maps for (include the whole string, including the `v` prefix)",
+				allowEmpty = false
+			)
+		}
+
+		steps {
+			mergeTrunk()
+			bashNodeScript {
+				name = "Upload source maps to Sentry"
+				scriptContent = """
+					# Install the Sentry CLI binary
+					curl -sL https://sentry.io/get-cli/ | bash
```

Comment:
We might be able to put this in one of our docker images instead of running it every time. For example, the wpcom image has a bunch of stuff installed in it: https://github.com/Automattic/wp-calypso/blob/29b2eb19eb032b1dbad22395e1cdb91611ce225c/Dockerfile.base#L128

Reply:
How often are those images built? 

I think this is a good idea, but I'll probably do it as a follow-up. I want to confirm it works well first :)

Reply:
the base image is rebuilt once a day, but we can always trigger the build manually to get a fresh copy